tasks/copy_to_glacier_cumulus_translator/copy_to_glacier_cumulus_translator.py

```diff
@@ -0,0 +1,86 @@
+from typing import Dict, Any, List
+
+import run_cumulus_task
+from cumulus_logger import CumulusLogger
+
+LOGGER = CumulusLogger()
+
+
+class ReformatRequestError(Exception):
+    """
+    Exception to be raised if the request fails in any way.
+    """
+
+
+def task(event: Dict[str, Any], context) -> List[Dict[str, Any]]:
+    """
+    Transforms event from Cumulus format to Orca copy_to_glacier format.
+    Args:
+        event: A dict with the following keys:
+            'input': See schemas/input.json
+            'config': See schemas/config.json
+        context: An object passed through by CMA. Unused.
+
+    Returns:
+        See schemas/output.json
+    """
+    # Cannot use f"" because of '{}' handling bug in CumulusLogger
```

Comment:
have we created  a card for fixing this bug for Cumulus team? @bhazuka

Reply:
I have not yet. Talking with the team on slack, we can submit a pull request and update the code ourselves for a release.